src/graphql/graphql.ts

```diff
@@ -1045,6 +1060,19 @@ function isV1ModelType(type: any): type is gql.GraphQLObjectType {
   );
 }
 
+function isV1EmbeddedType(type: any): type is gql.GraphQLObjectType {
+  return (
+    gql.isObjectType(type) &&
+    !some(type.getInterfaces(), (i) => i.name === 'Node') &&
+    type.name !== '__Schema' &&
```

Comment:
This is sad, but it works for our schema.

It was getting an infinite recursion on the embedded resolver until I filtered these out. If you have any ideas on how to improve, I'm all ears.

Reply:
@eskrm @ted-faros ^

Reply:
Thanks @eskrm !!